app/src/main/java/com/android/template/ui/home/HomeContract.kt

```diff
@@ -5,7 +5,11 @@ import com.android.template.base.ScopedPresenter
 
 class HomeContract private constructor() {
     interface Presenter : ScopedPresenter {
+        fun renderNavigationBar()
```

Comment:
`render` metode se obično definiraju na view-u. Probat ću skužiti iz koda koji je use case ovdje, ali slobodno pojasni ak buš imao vremena.

Reply:
ideja je da se u prezenteru pozove metoda koja provjerava ako je user logiran te onda ovisno o tome renderira navigation view
to mi treba jer kad je user logiran ima neke dodatne mogućnosti u navigation draweru pa samo stavim show/hide taj određeni submenu